docs/content/administration/_index.md

```diff
@@ -9,8 +9,8 @@ alwaysopen = false
 
 This app requires the following apps to be enabled:
 
-* [**Teams**](https://apps.nextcloud.com/apps/circles)
-* [**Text**](https://apps.nextcloud.com/apps/text)
+* **Teams** (with Nextcloud >= 30) or **Circles** (with Nextcloud <= 29)
```

Comment:
It will be teams from 29 on, right?
```suggestion
* **Teams** (with Nextcloud >= 29) or **Circles** (with Nextcloud <= 28)
```

Reply:
Hehe ouch, good catch. Fixed now. I also added a note that all of these apps are shipped and enabled with recent Nextcloud releases.